doc/api/deprecations.md

```diff
@@ -979,6 +979,15 @@ Type: Runtime
 This was an undocumented helper function not intended for use outside Node.js
 core and obsoleted by the removal of NPN (Next Protocol Negotiation) support.
 
+<a id="DEP00XX"></a>
+### DEP00XX: Buffer#(read|write) functions without offset
```

Comment:
If you want to deprecate this, that should also be noted in the `buffer.md` docs.

However, I would suggest not deprecating this, or at least splitting that out into a different PR?

Reply:
Splitting out to a separate PR would likely be best. It would, for instance, stop this from being a semver-major :-)

Reply:
Yes, +1 to splitting the doc-deprecation to a separate PR so we could review the fix separately and without debating on the doc-deprecation.

Reply:
I removed the deprecation as suggested.